docs/test_indexing.md

```diff
@@ -0,0 +1,10 @@
+1. Create a publishing file in Alma (Either in the sandbox or production)
+    1. Create a Query that contains the data you would like to query.  Save the query after you have it correct
```

Comment:
## Index one file

1. Create a data set in Alma based on a query search.
    1. .....